commit 032d645f1695545bf459e556223a2567a4a5db6a
Author: Yang Bo <yangbo@linuxdeepin.com>
Date:   Sun Jan 3 18:29:10 2016 +0800

    UBUNTU: SAUCE: (no-up) Added quirk to recognize GE0301 3G modem as an interface.
    
    Rejected by upstream, but continue to carry anyway.
    http://lkml.indiana.edu/hypermail/linux/kernel/1303.1/03158.html
    
    OriginalAuthor: Timo Aaltonen <tjaalton@ubuntu.com>
    BugLink: http://bugs.launchpad.net/bugs/348861
    
    Signed-off-by: Manoj Iyer <manoj.iyer@canonical.com>
    Signed-off-by: Tim Gardner <tim.gardner@canonical.com>
    Signed-off-by: Andy Whitcroft <apw@canonical.com>
    Acked-by: Tim Gardner <tim.gardner@canonical.com>
    Acked-by: Stefan Bader <stefan.bader@canonical.com>
    Signed-off-by: Yang Bo <yangbo@linuxdeepin.com>

diff --git a/drivers/usb/storage/unusual_devs.h b/drivers/usb/storage/unusual_devs.h
index 6b24791..a01e901 100644
--- a/drivers/usb/storage/unusual_devs.h
+++ b/drivers/usb/storage/unusual_devs.h
@@ -1262,6 +1262,13 @@ UNUSUAL_DEV(  0x0af0, 0x6971, 0x0000, 0x9999,
 		USB_SC_DEVICE, USB_PR_DEVICE, option_ms_init,
 		0),
 
+/* Reported by Timo Aaltonen <tjaalton@ubuntu.com> */
+UNUSUAL_DEV( 0x0af0, 0x7011, 0x0000, 0x9999,
+		"Option",
+		"Mass Storage",
+		USB_SC_DEVICE, USB_PR_DEVICE, option_ms_init,
+		0 ),
+
 /* Reported by F. Aben <f.aben@option.com>
  * This device (wrongly) has a vendor-specific device descriptor.
  * The entry is needed so usb-storage can bind to it's mass-storage
